vignettes/algorithm_data.Rmd

```diff
@@ -0,0 +1,151 @@
+---
+title: "Description of data objects related to the algorithm"
```

Comment:
```suggestion
title: "Description of data components related to the OSDC algorithm"
```
Not sure you if you refer to the same things with "data objects" and "data components"? :)